### PR TITLE
Move to Github Actions for CI

### PR DESCRIPTION
**Pull request by MariusRumpf**
- Adds a CI config for Github Actions
- Removes Appveyor and Travis as CI
- Cleans up badges in the readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,6 @@
 # LIFX LAN Node.js Library
 
 [![NPM Version](https://img.shields.io/npm/v/lifx-lan-client.svg)](https://www.npmjs.com/package/lifx-lan-client)
-[![Build Status](https://img.shields.io/travis/MariusRumpf/lifx-lan-client/master.svg)](https://travis-ci.org/MariusRumpf/lifx-lan-client)
-[![Build status](https://img.shields.io/appveyor/ci/MariusRumpf/lifx-lan-client/master.svg)](https://ci.appveyor.com/project/MariusRumpf/lifx-lan-client)
-[![Dependency Status](https://dependencyci.com/github/MariusRumpf/lifx-lan-client/badge)](https://dependencyci.com/github/MariusRumpf/lifx-lan-client)
-[![codecov.io](https://img.shields.io/codecov/c/github/MariusRumpf/lifx-lan-client/master.svg)](https://codecov.io/github/MariusRumpf/lifx-lan-client?branch=master)
-[![Gitter Chat](https://img.shields.io/gitter/room/lifx-lan-client/Lobby.svg)](https://gitter.im/lifx-lan-client/Lobby)
-
```
Comment (ristomatti) [inline README.md:9]: I wasn't aware we still had references to your profile. Sorry about this!

**Comment by ristomatti** [comment]
This is awesome. I've yet to have time or inspiration to dig into GitHub actions. Is there a way to verify if the setup works?

**Comment by ristomatti** [comment]
Are today's actions bringing an old project to an end, or would you perhaps be interested in joining the project? As an honorary member if nothing else? :slightly_smiling_face:

File: README.md (hunk 1 above)
Comment (MariusRumpf) [reply README.md:9]: No problem, I was just wondering about all the broken badges.

**Comment by MariusRumpf** [comment]
It is possible to see it in action in my fork to validate the results:
https://github.com/MariusRumpf/lifx-lan-client/actions/runs/4008683675

**Comment by MariusRumpf** [comment]
I am not sure how much time I can spare, but if I find some I will try to help out. Honorary member if nothing else sounds great 🙂 
Today marks the day I sunset the old project which I could not support any longer, alone.

**Review by ristomatti — APPROVED** [review]
LGTM!

**Comment by ristomatti** [comment]
I've had trouble finding time as well but I still have the incentive as most of our apartment is lit with LIFX ligs. I even bought some spares recently after spotting them for cheap on a local consumer electronics retailer, even while LIFX's EU store had all sold out.

Have you noticed LIFX brand has been sold? It's likely not good news. I have yet to see anything good come out of IoT brands getting sold. It was not at all surprising to me as the community forum has had almost zero activity for the past few years. A huge alarm bell was when the most visible and active LIFX employee [left the company](https://community.lifx.com/t/has-delfick-left-the-building/8318).

In fact I already had a bad feeling when they pushed out their, in my opinion, garbage rewrite of the mobile app. As a fan of the old simple app, I had a [few](https://community.lifx.com/t/lifx-app-4-0-beta-release/6052/3?u=ristomatti) [things](https://community.lifx.com/t/lifx-app-4-0-beta-release/6052/6?u=ristomatti) to [say](https://community.lifx.com/t/lifx-app-4-0-beta-release/6052/9?u=ristomatti) about the update.

While writing this, I also noticed the new owner of the brand is "reviewing our sales strategy in your region": :https://lifxshop.eu/

But now I'll end my rant and resume back to paid work :persevere:. Also, I've sent you an invite to join the team!

**Pull request merged**